CfStreamUploader/CfStreamUploader.Core.Test/CfStreamUploader.Core.Test.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Create a Folder "Tests" for all comming tests
CfStreamUploader/CfStreamUploader.Core.Test/**Tests**/CfStreamUploader.Core.Test.csproj